web/packages/design/src/ResourceIcon/icons.ts

```diff
@@ -207,6 +208,8 @@ import oktaLight from './assets/okta-light.svg';
 import onehundredonedomain from './assets/onehundredonedomain.svg';
 import onepasswordDark from './assets/onepassword-dark.svg';
 import onepasswordLight from './assets/onepassword-light.svg';
+import openaiDark from './assets/openai-dark.svg';
+import openaiLight from './assets/openai-light.svg';
```

Comment:
Are these used anywhere in Teleport anymore? Assist has been removed, so maybe we don't need these icons at all.

Reply:
It doesn't look like we use OpenAI icons anywhere else. I'll remove these icons and create a follow up PR to remove dead code.

Reply:
Thanks Bernard!